tools/src/test/resources/valid-evidence-1.3.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<bom serialNumber="urn:uuid:3e671687-395b-41f5-a30f-a58921a69b79" version="1" xmlns="http://cyclonedx.org/schema/bom/1.3">
+    <components>
+        <component type="application">
+            <group>com.google.code.findbugs</group>
+            <name>findbugs-project</name>
+            <version>3.0.0</version>
+            <licenses>
+                <license>
+                    <id>LGPL-3.0-or-later</id>
+                    <url>https://www.gnu.org/licenses/lgpl-3.0-standalone.html</url>
+                </license>
+            </licenses>
+            <purl>pkg:maven/com.google.code.findbugs/findbugs-project@3.0.0</purl>
+            <evidence>
+                <licenses>
+                    <license>
+                        <id>Apache-2.0</id>
+                        <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+                    </license>
+                    <license>
+                        <id>LGPL-2.1-only</id>
+                        <url>https://opensource.org/licenses/LGPL-2.1</url>
+                    </license>
+                </licenses>
+                <copyright>
+                    <text><![CDATA[Copyright 2012 Google Inc. All Rights Reserved.]]></text>
+                    <text><![CDATA[Copyright (C) 2004,2005 Dave Brosius <dbrosius@users.sourceforge.net>]]></text>
+                    <text><![CDATA[Copyright (C) 2005 William Pugh]]></text>
+                    <text><![CDATA[Copyright (C) 2004,2005 University of Maryland]]></text>
+                </copyright>
```

Comment:
Should these be represented more like this? Just thinking if we want to add anything to it in the future.

```
<copyrights>
  <copyright>
    <text>Copyright (c) 2000 Bleep</text>
  <copyright>
  <copyright>
    <text>Copyright (c) 2010 Bloop</text>
  <copyright>
</copyrights>
```

Reply:
Sure, they can me. It doesn't require any changes to the schema. But I put everything in CDATA because the second entry containing the email address will make the XML invalid without the CDATA.

Reply:
Sorry, ignore the cdata part, that was unintentional. I meant the xml element hierarchy: compyrights>copyright>text as opposed to copyrights>text

Reply:
I did it this way in the event we ever need to specify the file, line number, or any other type of data about where the copyright text was found. Not ideal, yes. But I think there may be some future room for improvement here in the event people start adopting it.

Reply:
I guess that stuff could be attributes on the text element. Yeah, never mind me :)